pennylane/templates/subroutines/interferometer.py

```diff
@@ -147,10 +151,12 @@ def Interferometer(theta, phi, varphi, wires, mesh="rectangular", beamsplitter="
                 # skip even or odd pairs depending on layer
                 if (l + k) % 2 != 1:
                     if beamsplitter == "clements":
-                        Rotation(phi[n], wires=[w1])
-                        Beamsplitter(theta[n], 0, wires=[w1, w2])
+                        Rotation(phi[n], wires=[w1])  # TODO: use select function ?
+                        Beamsplitter(theta[n], 0, wires=[w1, w2])  # TODO: use select function ?
```

Comment:
:thinking:

Reply:
I deleted those comments. They are flagging spots where the next PR will need to make changes.